src/constants/links.js

```diff
@@ -7,7 +7,7 @@ export const headerLinks = {
 };
 
 export const footerLinks = {
-  primary: ['link-one', 'link-two', 'link-three', 'link-four'],
+  primary: headerLinks.navDropDowns.flat().concat(headerLinks.parent),
```

Comment:
This file should be updated to use objects to store the label (from the translation files) along with the link, to make things more clear.

Reply:
> This file should be updated to use objects to store the label (from the translation files) along with the link, to make things more clear.

Yes, that will be addressed in a separate PR.

Reply:
> Also the key names in this file should correspond to the ones in the translation.json files, so it's easy to see how they relate

I can only think of renaming `headerLinks.navDropDowns` to something like `headerLinks.navDropDownSimpleLinks`. Where else do you see we can do this, or what were you thinking of?

Reply:
I think it depends on how the object is set up and if/how we rename the keys in the translation files

Reply:
Sure, we can consider this in the other PR I will create for objects.